op-node/rollup/attributes/attributes.go

```diff
@@ -36,8 +36,17 @@ type AttributesHandler struct {
 
 	attributes     *derive.AttributesWithParent
 	sentAttributes bool
+
+	EngDeriver EngDeriver
+}
+
+type EngDeriver interface {
```

Comment:
Can we move this interface to the engine package alongside EngDeriver and rename it something like `EngDeriverInterface` so that other dependencies on EngDeriver can find it easily?